### PR TITLE
Branch1

### PR DESCRIPTION
**Pull request by hencci**
Wrote 2 functions to get computer choice, and human choice

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+function getHumanChoice() {
+    let humanChoice = prompt("Rock, Paper, or Scissors?");
+    humanChoice = humanChoice.toUpperCase();
+
+    if (humanChoice === "ROCK" || humanChoice === "PAPER" || humanChoice === "SCISSORS") {
+        return humanChoice;
+    }
+    else {
+        alert("Invalid input. Try again");
```
Comment (Kadunapikin) [inline script.js]: I feel an alert message that'll let the person know they're to make selection from the provided options will be better than just Try again!

File: script.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    for (i = 0; i < 5; i++) {
+        const humanSelection = getHumanChoice();
+        const computerSelection = getComputerChoice();
+
+        playRound(humanSelection, computerSelection);
+    }
+    
+    console.log(`Human Score: ${humanScore} | Computer Score: ${cpuScore}`);
+    if (humanScore > cpuScore) {
+        console.log("You win the game! You're smarter then the computer")
```
Comment (Kadunapikin) [inline script.js]: Correct the spelling of then to than!

**Review by Kadunapikin — CHANGES_REQUESTED** [review]
Great work Henry, I'm really impressed. Make these few corrections and you're good! Happy coding!!!

File: script.js (hunk 1 above)
Comment (hencci) [reply script.js]: Done

File: script.js (hunk 2 above)
Comment (hencci) [reply script.js]: Done

**Pull request merged**